Bukkit/src/main/java/com/plotsquared/bukkit/listeners/PlayerEvents.java

```diff
@@ -2491,10 +2491,18 @@ public boolean entityDamage(Entity damager, Entity victim) {
                     return false;
                 }
             } else if (victim instanceof Creature) { // victim is animal
-                if (plot != null && (plot.getFlag(Flags.ANIMAL_ATTACK, false) || plot.getFlag(Flags.PVE, false) || plot
-                        .isAdded(plotPlayer.getUUID()))) {
-                    return true;
+                if (plot != null || plot.isAdded(plotPlayer.getUUID())) {
```

Comment:
This changes the logic. The removed snippet requires that the plot isn't null AND that the player is added to the plot, whereas this line would terminate if the plot isn't null, if the plot is null it would check if the player is added and thus result in a NullPointerException.